core/arch/arm/tee/entry_std.c

```diff
@@ -566,6 +567,10 @@ uint32_t __tee_entry_std(struct optee_msg_arg *arg, uint32_t num_params)
 		rv = OPTEE_SMC_RETURN_EBADCMD;
 	}
 
+	if (arg->entropy)
+		crypto_rng_add_event(CRYPTO_RNG_SRC_JITTER_RPC,
```

Comment:
We need to define a new source id in enum crypto_rng_src and use that instead.

Reply:
ok